src/components/dataset/dataset-summary.vue

```diff
@@ -0,0 +1,78 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <summary-item v-if="datasetDiff.length > 0" icon="link">
```

Comment:
This seems to be working as-is, but what do you think about making the `v-if` more explicit by specifying:

```js
datasetDiff.dataExists && datasetDiff.length > 0
```